network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/airtable_block.html

```diff
@@ -1 +1 @@
-<script src="https://static.airtable.com/js/embed/embed_snippet_v1.js"></script><iframe class="airtable-embed airtable-dynamic-height" src="{{ value.url }}" frameborder="0" onmousewheel="" width="100%" height="{{ value.height }}" style="background: transparent; border: 1px solid #ccc;"></iframe>
+<script src="https://static.airtable.com/js/embed/embed_snippet_v1.js"></script><iframe class="airtable-embed mb-4 airtable-dynamic-height" src="{{ value.url }}" frameborder="0" onmousewheel="" width="100%" height="{{ value.height }}" style="background: transparent; border: 1px solid #ccc;"></iframe>
```

Comment:
If we're touching this, let's make sure to also make this an [asynchronous](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-async), and  [deferred](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-defer) script load:
```suggestion
<script async defer src="https://static.airtable.com/js/embed/embed_snippet_v1.js"></script><iframe class="airtable-embed mb-4 airtable-dynamic-height" src="{{ value.url }}" frameborder="0" onmousewheel="" width="100%" height="{{ value.height }}" style="background: transparent; border: 1px solid #ccc;"></iframe>
```